test/validate/linux_validate.go

```diff
@@ -230,3 +322,39 @@ func (v *LinuxValidator) validate(stateFileIps stateFileIpsFunc, cmd []string, c
 	log.Printf("State file validation for %s passed", checkType)
 	return nil
 }
+
+func (v *LinuxValidator) ValidateDualStackNodeProperties() error {
+	log.Print("Validating Dualstack Overlay Linux Node properties")
+	nodes, err := k8sutils.GetNodeList(v.ctx, v.clientset)
+	if err != nil {
+		return errors.Wrapf(err, "failed to get node list")
+	}
+
+	for index := range nodes.Items {
+		nodeName := nodes.Items[index].ObjectMeta.Name
+		// check node status
+		nodeConditions := nodes.Items[index].Status.Conditions
+		if nodeConditions[len(nodeConditions)-1].Type != corev1.NodeReady {
+			return errors.Wrapf(err, "node %s status is not ready", nodeName)
```

Comment:
Is this because of error ? Or we expect the node to be ready after a certain time. ?
If it is going to be ready after a certain time, should we add a retry ?